nifipy/components.py

```diff
@@ -64,9 +70,15 @@ def get_controller_services(self, process_group = "root"):
     def _get(self, url):
         return requests.get(url)
 
-    def _post(self, url, data):
+    def _put(self, url, data):
         return requests.put(url, data=data, headers={"Content-Type": "application/json"})
- 
+
+    def _post(self, url, data):
+        return requests.post(url, data=data, headers={"Content-Type": "application/json", 'Connection': 'keep-alive'})
```

Comment:
I'm not super strict about this, but it's nice to have a look at PEP-8 in general.
Specifically in this case: Max length of a line should be 79 characters (https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
I will consider this in the future. Seems like there are several violations of this style.